specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -4141,6 +4141,18 @@
         "tableOption": {
           "type": "object",
           "description": "The option to handle sink table, such as autoCreate. For now only 'autoCreate' value is supported. Type: string (or Expression with resultType string)."
+        },
+        "sqlWriterUseTableLock": {
+          "type": "object",
+          "description": "Whether to use table lock during bulk copy. Type: boolean (or Expression with resultType boolean)."
+        },
+        "writeBehavior": {
+          "type": "object"
```

Comment:
Please add description to this field.

Reply:
BTW, it seems the definition `SqlDWWriteBehaviorEnum` is to limit the possible values for this field.
@jianyexi can we defined the field with the enum, but leaving `type: object` blank? In this case, the field should be either enumed `string` or `object`.

Reply:
cc @shawnxzq for swagger enum practice.

Reply:
@yxi339 The type:object feature is in plan, but not supported yet currently. So resolve the comment.